add html_in _advantages

olkarpova opened the pull request:
markup for advantages, img 1x 2x

Denis071194 reviewed (changes requested):
Змініть назви класів які у вас через пробіл поставте через дефіс бо у вас advantages about, а треба advantages-about.
Усі <img class=""> — порожні. Додайте класи.
Рекомендую додати потрібний клас, наприклад: <img class="advantages-img".
Кожен li має логічну структуру: <picture>, <h3>, <p> — це чітко, семантично і правильно.
alt-описи в зображеннях написані грамотно — тут лайк👍
Текст “Переваги” має бути підзаголовком. Треба так: <h3 class="advantages-about">Переваги</h3>
В принципі все добре внесіть правки які я написав і приступайте до стилізації. Гарного вам дня)

olkarpova commented:
I fixed all your comments and finished all styles for my section for mob tab and desk. Please note that  I changed padding in container class for tab and desk (according to figma)  which is global for all web site.

olkarpova commented:
I added margin for div class: advantages-thumb and small fixed for h3

Denis071194 reviewed (approved):
Виглядає все круто))
